### PR TITLE
Issue 126 master speedup

### PR DESCRIPTION
**Pull request by dougli1sqrd**
This is for https://github.com/geneontology/pipeline/issues/126

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -373,12 +386,23 @@ pipeline {
...
+    				retry(3){
+    				    sh 'make prepare_release'
+    				}
+    			    }
+			} else {
+			    // Otherwise we /are/ reusing, and we just grab the whole ontology directory on the current release,
+			    // and plop it in place, skipping the build.
+			    sh 'wget --recursive --no-parent  -R index.html* -nH  http://current.geneontology.org/ontology/'
+			    sh 'mv ontology/* ./src/ontology/target/'
+			    sh 'rm -r ontology'
```
Comment (kltm) [inline Jenkinsfile]: rmdir?

**Comment by kltm** [comment]
For #129 ; looks good to me.
We'll hold until tRNA and `release` are done.

File: Jenkinsfile (hunk 1 above)
Comment (dougli1sqrd) [reply Jenkinsfile]: sure

**Comment by kltm** [comment]
@dougli1sqrd Apologies--is this still a thing?

**Comment by dougli1sqrd** [comment]
We at one time thought about making master run faster. This is so out of sync at this time though that we could just cancel this and revist later if we wanted to.

**Comment by kltm** [comment]
I think we'll probably take a different approach if it comes to this again, especially after some of the work that's plated.

**Pull request closed without merging**